samples/cellular/nrf_cloud_multi_service/overlay_nrf7002ek_wifi_coap_no_lte.conf

```diff
@@ -76,6 +76,16 @@ CONFIG_DATE_TIME_MODEM=n
 CONFIG_DATE_TIME_NTP=y
 CONFIG_DATE_TIME_LOG_LEVEL_DBG=y
 
+## Enable date_time retries for Wi-Fi
+## This prevents occasional DNS lookup flukes from
+## rendering the device unable to connect until the next
+## DATE_TIME_UPDATE_INTERVAL_SECONDS elapses (default 4 hours)
```

Comment:
Nit: I'd say that let's add `CONFIG_` prefix in all comments outside the Kconfig file.

```suggestion
## CONFIG_DATE_TIME_UPDATE_INTERVAL_SECONDS elapses (default 4 hours)
```